packages/cli/src/commands/sandbox/logs.ts

```diff
@@ -0,0 +1,173 @@
+      if (!sandbox) {
+        isRunning = false
+        break
+      }
+      await wait(5000)
+    }
+  }
+
+  monitor()
+
+  return () => isRunning
+}
+
+export const logsCommand = new commander.Command('logs')
+  .description('show logs for sandbox')
+  .argument('<sandboxID>', `show longs for sandbox specified by ${asBold('<sandboxID>')}`)
+  .alias('lg')
+  .action(async (sandboxID: string) => {
+    try {
+      const apiKey = ensureAPIKey()
+      const getIsRunning = waitForSandboxEnd(apiKey, sandboxID)
+
+      let start: number | undefined
+      let isFirstRun = true
+      let firstLogsPrinted = false
+
+      console.log(`\nLogs for sandbox ${asBold(sandboxID)}:`)
+
+      const isRunningPromise = listSandboxes({ apiKey }).then(r => r.find(s => getLongID(s.sandboxID, s.clientID) === sandboxID)).then(s => !!s)
+
+      do {
+        try {
+          const logs = await listSandboxLogs({ apiKey, sandboxID, start })
+
+          if (logs.length !== 0 && firstLogsPrinted === false) {
+            firstLogsPrinted = true
+            process.stdout.write('\n')
+          }
+
+          for (const log of logs) {
+            printLog(log.timestamp, log.line)
+          }
+
+          const isRunning = await isRunningPromise
+
+          if (!isRunning && logs.length === 0 && isFirstRun) {
+            console.log(`\nStopped printing logs — sandbox ${withUnderline('not found')}`)
+            break
+          }
+
+          if (!isRunning) {
+            console.log(`\nStopped printing logs — sandbox is ${withUnderline('closed')}`)
+            break
+          }
+
+          const lastLog = logs.length > 0 ? logs[logs.length - 1] : undefined
+          if (lastLog) {
+            start = new Date(lastLog.timestamp).getTime() + 1
+          }
+        } catch (e) {
+          if (e instanceof getSandboxLogs.Error) {
+            const error = e.getActualType()
+            if (error.status === 401) {
+              throw new Error(
+                `Error getting sandbox logs - (${error.status}) bad request: ${error}`,
+              )
+            }
+            if (error.status === 404) {
+              throw new Error(
+                `Error getting sandbox logs - (${error.status}) not found: ${error}`,
+              )
+            }
+            if (error.status === 500) {
+              throw new Error(
+                `Error getting sandbox logs - (${error.status}) server error: ${error}`,
+              )
+            }
+          }
+          throw e
+        }
+
+        await wait(400)
+        isFirstRun = false
+      } while (getIsRunning())
+    } catch (err: any) {
+      console.error(err)
+      process.exit(1)
+    }
+  })
+
+function printLog(timestamp: string, line: string) {
+  const log = JSON.parse(line)
+
+  const time = `[${new Date(timestamp).toISOString().replace(/T/, ' ').replace(/\..+/, '')}]`
+  let level = log['level'].toUpperCase()
```

Comment:
would be nice to have all the levels with the same width, so you have all the logs aligned